### ISSUE TITLE
[FEATURE] Saving live videos

### ISSUE THREAD

**Issue by ppp789** [eni9889/IG-PP-Issues#101]
When the live video is end, you can watch it again, can you add download?

**Comment by dimoca** [comment]
Hi, is it possible to add a button to download saved live streams?

**Comment by ppp789** [comment]
Yes, when the stream end, you have 24 hours to watch that, its replay

**Comment by dimoca** [comment]
Yes but i wanna can download this...
